dags/shredder.py

```diff
@@ -82,7 +82,7 @@
 ]
 common_task_args = {
     "image": docker_image,
-    "is_delete_operator_pod": True,
+    "on_finish_action": OnFinishAction.DELETE_POD.value,
```

Comment:
Shouldn't this work without the `.value`?

Reply:
Code is
```
elif on_finish_action is not None:
    kwargs["on_finish_action"] = OnFinishAction(on_finish_action)
```
I think `OnFinishAction(on_finish_action)` resolves correctly if you give it the enum object

Reply:
While passing an `OnFinishAction` enum would indeed currently work (and as you noticed `GKEStartPodOperator` is itself passing an `OnFinishAction` enum to the `KubernetesPodOperator` constructor), the `GKEStartPodOperator` and `KubernetesPodOperator` constructors type hint their `on_finish_action` arguments as `str | None` and `str`, respectively, so I figured it was safer to abide by that and not assume passing an `OnFinishAction` enum would continue to work.